tests/main/snapd-apparmor/task.yaml

```diff
@@ -0,0 +1,73 @@
+summary: Ensure snapd-apparmor works as expected
```

Comment:
Perhaps we should have a variant of the test that executes in LXD as well?

Reply:
This sounds like a good idea, I'll see if I can come up with something.